src/components/map/Popups/StudyPopup.tsx

```diff
@@ -52,30 +51,39 @@ function getTranslatedPopulationGroup(popGroupOptions: Record<string, string>[],
 
 export default function StudyPopup(record: AirtableRecord, popGroupOptions: Record<string, string>[]) {
     return (
-        <div className="popup-content" >
-            <div className={"d-flex justify-content-between mb-1"}>
+        <div className="popup-content pt-2" >
+            {/*Header section*/}
+            <div className={"popup-section"}>
                 <div className="popup-title">
                     {Translate(`${record.estimate_grade}StudyDetails`)}
                 </div>
                 <div className="popup-subtitle">
-                    {getGeography(record.city, record.state, record.country)}
+                    {record.url ? <a href={record.url} target="_blank" rel="noopener noreferrer">{record.source_name}</a> : record.source_name}
                 </div>
             </div>
-            <Divider />
-            {row(Translate("StudyName"), record.url ? <a href={record.url} target="_blank" rel="noopener noreferrer">{record.source_name}</a> : record.source_name)}
-            {/*{row("Scope", record.estimate_grade)}*/}
```

Comment:
Should these comments be removed?

Reply:
yeee cause they were the old way that this stuff was written, I changed the way the data is presented but all this data is there so these comments do not add any value